test-scaling/script.sh

```diff
@@ -0,0 +1,123 @@
+#! /usr/bin/env bash
+
+registry='francisregistry.azurecr.io'
+image_tag='francis-test-scaling'
```

Comment:
If it's in the tree - the user must be able to run it.

Also, having "test-scaling" in the root is not ideal, as we'll have different types of tests.
tests/scaling-nodes?